.github/workflows/update-native-sdks.yml

```diff
@@ -0,0 +1,83 @@
+# Bump Native SDK version numbers whenever a repository dispatch of a release
+# is received
+name: Bump Native SDKs
+on:
+  repository_dispatch:
+    types: ['ios-sdk-release', 'android-sdk-release']
+jobs:
+  update_android_sdk_version:
+    runs-on: ubuntu-latest
+    if: github.event.client_payload.platform == "android"
```

Comment:
I think android sdk repo might be missing this right now

Reply:
`client-payload` (not underscore), right?

Reply:
It is underscore. It's one of those annoying instances where the workflow we're using uses hyphens, but the resultant payload is underscores: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#repository_dispatch

Reply:
annoying